### ISSUE TITLE
SS-Plex Error after update to PMS V1.6.1.3722 

### ISSUE THREAD

**Issue by emandino** [mikew/ss-plex.bundle#66]
Hello Mike,

Finished updating PMS to V1.6.1.3722 and restarted the server. Unfortunately, SS-Plex failed to start correctly. Attempted to re-install the latest version thru WebTools UAS but received no response. No Logs for SS-Plex were generated. Attached please find a screenshot of the channel's landing page.

Thanks Edwin

![screen shot 2017-05-17 at 2 45 18 pm](https://cloud.githubusercontent.com/assets/4790369/26178126/813b6a80-3b11-11e7-8802-3b6c9669a6dc.png)

**Comment by crhatfield** [comment]
Make sure you update from github, the old unofficial app store (aka web tools) version has been broke for a couple years; last I knew.. Now I did have a "Whoops" error for about an hour or so last night, that was probably happening to everyone and just a coincidence. Are you still getting that error?

For my own sanity, I just checked and I'm running that version of Plex Server and SSP is working fine for me.
